packages/ses/src/module-instance.js

```diff
@@ -278,14 +290,12 @@ export const makeModuleInstance = (
     // The module exports namespace will be wrapped in a module namespace
     // exports proxy which will serve as a "module exports namespace exotic
     // object".
-    keys(moduleProps)
+    keys(exportsProps)
       .sort()
```

Comment:
Mention in comment that normally `sort`ing property names doesn't work because symbols don't sort. But here it is ok because we can only encounter string-named (in fact identifier-string-named) properties.

Reply:
```
    // Sorting properties is not generally reliable because some properties may
    // be symbols, and symbols do not have an inherent relative order, but
    // since all properties of the exports namespace must be keyed by a string
    // and the string must correspond to a valid identifier, sorting these
    // properties works for this specific case.
```